test/data/golog_app/robot_camera_spec.pbtxt

```diff
@@ -0,0 +1,46 @@
+disjunction {
+		disjuncts {
+			finally {
+				formula {
+					conjunction {
+						conjuncts {
+							negation {
+								formula { atomic { symbol: "camera_on()"}}
+							}
+						}
+						conjuncts {
+							atomic { symbol: "grasping()"}
+						}
+					}
+				}
+			}
+		}
+		disjuncts {
+			finally {
+				formula {
+					conjunction {
+						conjuncts {
+							negation {
+								formula { atomic { symbol: "camera_on()"}}
+							}
+						}
+						conjuncts {
+							finally {
+								formula { atomic { symbol: "grasping()"}}
+								interval {
+									lower {
+										value: 0
+										bound_type: WEAK
+									}
+									upper {
+										value: 2
+										bound_type: WEAK
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	}
```

Comment:
Better only indent with two spaces, like in the other spec file:
https://github.com/morxa/tacos/blob/main/examples/railroad/spec.pbtxt

Reply:
Also, there is a missing newline at the end